mitosheet/mitosheet/types.py

```diff
@@ -148,17 +151,41 @@ class CodeSnippetEnvVars(TypedDict):
         MITO_CONFIG_CODE_SNIPPETS_URL: str
         MITO_CONFIG_CODE_SNIPPETS_SUPPORT_EMAIL: Optional[str]
 
+    class ParserMatch(TypedDict):
+        type: str # 'column header' | 'index label'
```

Comment:
In the parser, we set the `type` to different values (below). Want to update this comment?

```
COLUMN_HEADER_MATCH_TYPE = 'column header match type'
INDEX_LABEL_MATCH_TYPE = 'index label match type'
```